### PR TITLE
Flockfile misc refactorings

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Some small (misc) refactorings in the `flockfile` crate.

Only readability things, nothing too important.

### CODE REVIEW

File: crates/common/flockfile/src/unix.rs
Comment (didier-wenzek) [inline crates/common/flockfile/src/unix.rs:64]: Nitpicking. Just to be consistent with the following `map_err` and highlight that the `source` is actually an error.

```suggestion
            .map_err(|err| FlockfileError::FromIo {
                path: path.clone(),
                source: err,
            })?;
```

**Review by didier-wenzek — APPROVED** [review]

File: crates/common/flockfile/src/unix.rs
Comment (matthiasbeyer) [reply crates/common/flockfile/src/unix.rs]: Fixed in the fixup commit (will squash once approved).

Btw: You are really welcome to use the use the "Request changes" review option in cases like this and block merging with it. Even (or maybe _especially_) when it is nitpicking!

**Comment by didier-wenzek** [comment]
> Btw: You are really welcome to use the use the "Request changes" review option in cases like this and block merging with it. Even (or maybe especially) when it is nitpicking!

I try to make a difference between "I would have done thing differently" and "I cannot accept this".

**Comment by didier-wenzek** [comment]
Will merge once the --fixup squashed.

**Pull request merged**
